packages/cli/src/data/schema/graph.js

```diff
@@ -1,5 +1,78 @@
 const { gql } = require('apollo-server-express');
 
+const getMenuFromGraph = async (root, { pathname, filter = '', orderBy = '' }, context) => {
+  const { graph } = context;
+  let items = [];
+
+  graph
```

Comment:
Still would like to explore how custom front matter could allow for more flexibility from userland space, so not sure if this kind of logic should happen in the CLI or in the user's own component.